### DISCUSSION TITLE
hide toolbar buttons

### DISCUSSION THREAD

**Discussion by hellhorse123** [epam/ketcher#5879] [Q&A] [answered]
Is there a way to hide some buttons on the toolbar via Editor options?

**Comment by rrodionov91** [comment, marked as answer]
Hi @hellhorse123 

There is "buttons" property in Editor component:
<img width="542" alt="image" src="https://github.com/user-attachments/assets/d89eab33-a661-4aa2-9652-400ebc9ed93b" />

**Reply by rrodionov91** [reply]
Example 
```
buttons={{ save: { hidden: true } }}
```

**Comment by rbgros1** [comment]
I have the same question. Below, I have pasted the code that I am using to initiate a ketcher canvas and the editor associated with it. Now I want to hide the Add image button on the western toolbar and the Switch to macromolecule mode on the northern toolbar.   How do I use this editor to hide buttons? 

```
var ketcherInstances = [];

function initKetcher(appletName) {
    const ketcherFrame = document.getElementById(appletName);
    if ('contentDocument' in ketcherFrame) {
        var contentWindow = ketcherFrame.contentWindow;
        if (contentWindow && contentWindow.ketcher) {
            ketcherInstances[appletName] = contentWindow.ketcher;
            ketcherInstances[appletName].editor.buttons = {
                { addImage: { hidden: true } }
            };
        } else {
            // wait, try again
            setTimeout('initKetcher("' + appletName + '");', 200);
        }
    } else {
        alert('Cannot find content in iframe; aborting.');
    } // if there's a ketcher in the iframe
} // initKetcher()

```
